shared/settings/nav/index.desktop.js

```diff
@@ -56,6 +56,7 @@ const variantBannerStyle = {
 const itemStyle = {
   ...globalStyles.flexBoxRow,
   height: 40,
+  paddingRight: globalMargins.small,
```

Comment:
Is there a reason this right spacing isn't expressed as a margin [here](https://github.com/keybase/client/pull/4293/commits/b2999f6b4e386f38e0035e68204d6c62de894b7d#diff-f372159314e91f2152299b92b8bb72a5L19) to match the left spacing?